package.json

```diff
@@ -175,6 +182,7 @@
     "monaco-editor-webpack-plugin": "^1.8.2",
     "monaco-languageclient": "^0.11.0",
     "monaco-textmate": "^3.0.1",
+    "node-fetch": "^2.6.1",
```

Comment:
not sure if this will get removed with tree shaking, but this really isnt a dependency of our application. only a dependency for lighthouse.  perhaps lighthouse should be treated as its own application and have its own package.json?

Reply:
ah. good point. or should it be a dev dep?

Reply:
moved to new package.json.